core/src/alpenglow_consensus/voting_loop.rs

```diff
@@ -1008,14 +1006,7 @@ impl VotingLoop {
             bank.last_blockhash(),
         );
 
-        if !context.has_new_vote_been_rooted {
-            context.voted_signatures.push(vote_tx.signatures[0]);
-            if context.voted_signatures.len() > MAX_VOTE_SIGNATURES {
-                context.voted_signatures.remove(0);
-            }
-        } else {
-            context.voted_signatures.clear();
-        }
+        // No-op: we don't store voted_signatures anymore
```

Comment:
Remove this comment, and lets also remove `has_new_vote_been_rooted` and `voted_signatures` from the voting context